Client/src/Components/Label/index.jsx

```diff
@@ -116,13 +116,14 @@ ColoredLabel.propTypes = {
  * @param {Object} props
  * @param { 'up' | 'down' | 'cannot resolve'} props.status - The status for the label
  * @param {string} props.text - The text of the label
+ * @param {boolean} props.hasDot - Whether to show a dot or not
  * @returns {JSX.Element}
  * @example
  * // Render an active label
  * <StatusLabel status="up" text="Active" />
  */
 
-const StatusLabel = ({ status, text, customStyles }) => {
+const StatusLabel = ({ status, text, hasDot = true, customStyles }) => {
```

Comment:
Hey, Rogerio! Thanks for all you patience and effort to contribute to this project. 
Happy everything is working now. 
I worry about the hasDot property, though. It would allow someone to pass a dot to a number status, which does not exist in the design system. 

I see two ways out of this:
1) allowing status or httpStatus as props. I think something like this would type the props accordingly
const baseProps = {
	text: PropTypes.string.isRequired,
	customStyles: PropTypes.object,
};

const statusShape = PropTypes.shape({
	status: PropTypes.oneOf(["up", "down", "paused", "pending"]).isRequired,
	...baseProps,
});

const httpStatusShape = PropTypes.shape({
	httpStatus: PropTypes.oneOf(["400", "500", "cannot resolve"]).isRequired,
	...baseProps,
});

StatusLabel.propTypes = {
	statusProps: PropTypes.oneOfType([statusShape, httpStatusShape]).isRequired,
};

This way we us httpStatus code as proxy to know if we are dealing with status or httpStatus, and apply the dot accordingly. 

2) Get back to using two components. StatusLabel and HttpStatusLabel would both implement the BaseLabel.
I believe this approach is better.

In either approach, I suggest keeping the getGenericStatus function inside of the component.

Reply:
@rog404 @marcelluscaio Agreed, why don't we refactor out `BaseLabel` from the `Label` component and have both `StatusLabel` and `HttpStatusLabel` extend that.

That should keep confusion to a minumum and make use of the components very straightforward.

Appreciate you going back and forth on this one @rog404 , sometimes it takes some discussion to tease out the best approach :joy:

Reply:
Don't worry! I believe a good PR strategy should be just like this—well thought-out and full of great ideas. As soon as I have some time, I'll start working on this refactoring.

Reply:
> Don't worry! I believe a good PR strategy should be just like this—well thought-out and full of great ideas. As soon as I have some time, I'll start working on this refactoring.
> 

Absolutely, good results come from good discussions 👌Looking forward to seeing what you build!